tools/codegen/codespec/string_case.go

```diff
@@ -0,0 +1,26 @@
+package codespec
+
+import (
+	"strings"
+)
+
+type SnakeCaseString string
+
+func (snake SnakeCaseString) SnakeCase() string {
+	return string(snake)
+}
+
+func (snake SnakeCaseString) PascalCase() string {
```

Comment:
knit: this is used by our mock library, but I guess you already have everything you need: https://pkg.go.dev/github.com/huandu/xstrings#ToSnakeCase

via https://vektra.github.io/mockery/latest/configuration/#functions